cocotb/share/lib/vhpi/VhpiCbHdl.cpp

```diff
@@ -898,14 +898,14 @@ int VhpiTimedCbHdl::cleanup_callback()
 VhpiReadwriteCbHdl::VhpiReadwriteCbHdl(GpiImplInterface *impl) : GpiCbHdl(impl),
                                                                  VhpiCbHdl(impl)
 {
-    cb_data.reason = vhpiCbRepEndOfProcesses;
+    cb_data.reason = vhpiCbRepLastKnownDeltaCycle;
     cb_data.time = &vhpi_time;
 }
 
 VhpiReadOnlyCbHdl::VhpiReadOnlyCbHdl(GpiImplInterface *impl) : GpiCbHdl(impl),
                                                                VhpiCbHdl(impl)
 {
-    cb_data.reason = vhpiCbRepLastKnownDeltaCycle;
+    cb_data.reason = vhpiCbRepEndOfTimeStep;
```

Comment:
These changes are #1498, It would be nice to finish that table (needs testing with Cadence simulators) before we pull the trigger on this change.